### PR TITLE
add payout account crud operations

### PR DESCRIPTION
**Pull request by tuncaserhat**

### CODE REVIEW

File: adapter/model.go [hunk 1]
```diff
@@ -685,6 +717,15 @@ type ApmDepositPaymentResponse struct {
 	WalletTransaction   *WalletTransaction   `json:"walletTransaction"`
 }
 
+type PayoutAccountResponse struct {
+	Id                  *int64             `json:"id"`
```
Comment (Selahattinn) [inline adapter/model.go:721]: why this fields are pointer?

File: tests/settlement_test.go [hunk 2]
```diff
@@ -1,22 +1,90 @@
...
 
 import (
-    "context"
-    "github.com/craftgate/craftgate-go-client/adapter"
-    craftgate "github.com/craftgate/craftgate-go-client/adapter"
-    "github.com/davecgh/go-spew/spew"
-    "testing"
+	"context"
+	"github.com/craftgate/craftgate-go-client/adapter"
+	craftgate "github.com/craftgate/craftgate-go-client/adapter"
```
Comment (Selahattinn) [inline tests/settlement_test.go:6]: import order can be change to standart

File: adapter/settlement.go [hunk 3]
```diff
@@ -23,3 +24,63 @@ func (api *Settlement) CreateInstantWalletSettlement(ctx context.Context, reques
 
 	return response.Data, nil
 }
+
+func (api *Settlement) CreatePayoutAccount(ctx context.Context, request CreatePayoutAccountRequest) (*PayoutAccountResponse, error) {
```
Comment (Selahattinn) [inline adapter/settlement.go:28]: you can rename CreatePayoutAccountRequest -> CreatePayoutAccountRequestBody

File: adapter/model.go (hunk 1 above)
Comment (tuncaserhat) [reply adapter/model.go:721]: Yes, most of these fields are not nullable, I will remove pointers.

File: adapter/settlement.go (hunk 3 above)
Comment (tuncaserhat) [reply adapter/settlement.go:28]: We have a naming convention for request types, so it should end with Request

File: tests/settlement_test.go (hunk 2 above)
Comment (Selahattinn) [reply tests/settlement_test.go:6]:
```
import (
	"context"
	"testing"
	
	"github.com/davecgh/go-spew/spew"
	
	craftgate "github.com/craftgate/craftgate-go-client/adapter"
)
```

I said that this format is the most used format. You can also use the -local flag if you want. No big deal just to get close to the standard

**Review by abalikci — APPROVED** [review]

**Pull request merged**